dat/missions/dvaered/colonel-transport.lua

```diff
@@ -0,0 +1,165 @@
+--[[
+<?xml version='1.0' encoding='utf8'?>
+<mission name="Dvaered Colonel Escort">
+ <unique />
+ <priority>4</priority>
+ <chance>43</chance>
+ <location>Bar</location>
+ <faction>Dvaered</faction>
+ <cond>require("misn_test").reweight_active()</cond>
+</mission>
+--]]
+--[[
+
+   Mission: Escort a Dvaered colonel
+   Description: Small mission where you escort a Dvaered Arsenal.
+                This is a one-off that's not part of any major
+                storyline; the in-game purpose will remain a mystery.
+
+--]]
+
+local escort = require "escort"
+local fmt = require "format"
+local lmisn = require "lmisn"
+local neu = require "neutral"
+local vn = require "vn"
+local vni = require "vnimage"
+
+local reward = 750e3
+
+local source_system = nil
+
+local ffriendly = nil -- codespell:ignore ffriendly
+local fhostile = nil
+
+local npc_name = _("Dvaered Colonel")
+local npc_portrait = nil
+local npc_image = nil
+
+local m = vn.newCharacter( npc_name, {image=npc_image} )
+
+function create()
+   local spbs = lmisn.getSpobAtDistance( nil, 2, 6, "Dvaered" )
+   if #spbs <= 0 then misn.finish(false) end
+   mem.destspb = spbs[ rnd.rnd(1,#spbs) ]
+
+   mem.npc_image, mem.npc_portrait = vni.dvaeredMilitary()
+   misn.setNPC(npc_name, npc_portrait, _("A Dvaered, very professional-looking, is sitting with an excellant posture at the bar.") )
+end
+
```

Comment:
```suggestion
local function factions()
   local dvaered_factions = faction.get("Dvaered")

   local ffriendly = faction.dynAdd( dvaered_factions, "radver", _("Dvaered") ) -- codespell:ignore ffriendly
   local fhostile = faction.dynAdd( dvaered_factions, "radver_baddie", _("Dvaered Warlord") )
   faction.dynEnemy( ffriendly, fhostile ) -- codespell:ignore ffriendly
   return ffriendly, fhostile -- codespell:ignore ffriendly
end
   ```
   
   Define the faction generation function here.